LiveStream/Models/LiveStreamEvent.swift

```diff
@@ -65,37 +73,39 @@ public func == (lhs: LiveStreamEvent, rhs: LiveStreamEvent) -> Bool {
 extension LiveStreamEvent: Decodable {
   static public func decode(_ json: JSON) -> Decoded<LiveStreamEvent> {
     let create = curry(LiveStreamEvent.init)
-    return create
-      <^> json <| "creator"
-      <*> json <| "firebase"
+
+    let tmp1 = create
```

Comment:
here's where im doing the wild json decoding in order to accommodate the fact that we get two very different json representations depending on the api endpoint. I've written two huge tests here https://github.com/kickstarter/ios-oss/pull/53/files#diff-22bc6046f929d51b08d91fe1c7ac7e89 where i basically took a snapshot of json from the api in each case and verify that they decode correctly.